### DISCUSSION TITLE
Trying to install Shark

### DISCUSSION THREAD

**Discussion by Arcadia245** [nod-ai/SHARK#764] [Q&A] [answered]
Hey everyone, 

When I try to install charge from the exe I get this error:

torchvision\io\image.py:13: UserWarning: Failed to load image Python extension:
torch\_jit_internal.py:839: UserWarning: Unable to retrieve source for @torch.jit._overload function: <function _DenseLayer.forward at 0x0000016D28D6DA20>.
  warnings.warn(
torch\_jit_internal.py:839: UserWarning: Unable to retrieve source for @torch.jit._overload function: <function _DenseLayer.forward at 0x0000016D28D6FD00>.
  warnings.warn(
vulkan devices are available.
cuda devices are not available.
Running on local URL:  http://0.0.0.0:8080

Not sure what to do. I apologize, I'm not a coder or anything and I have no idea about github. I just wanted to try stable diffusion on my amd card.

Thanks to anyone who can help!

**Comment by powderluv** [comment, marked as answer]
Can you please let us know what card? You will need the specific AMD driver at http://shark.sd

**Reply by Arcadia245** [reply]
Hi, I use a 5700 xt. I installed those drivers and finally got it to work. Though I still get all the errors above. I guess it doesn't matter. Thanks for your help!
